Update read_data.py

varshini2305 opened the pull request:
processing the contents after detecting the format as JSON - to represent the contents of JSON file in the form of numpy array for further processing

hornig committed:
adds positional_observation_reporting, fixes lagrange parameters

hornig committed:
adds first rudimentary identification of csv, json and iod.

hornig committed:
adds detection of csv delimiter

hornig committed:
fixes problem to use positional obs reporting from main

the absolute and relative paths can be used like this.
now the function can be used from other subfolders.

hornig committed:
adds same pathing to all

varshini2305 committed:
Update read_data.py

varshini2305 committed:
Add files via upload

varshini2305 committed:
Update read_data.py

varshini2305 committed:
Update positional_observation_reporting.py

varshini2305 committed:
Merge pull request #1 from varshini2305/ukformatdetect

Detect UK file format

hornig commented:
hey @varshini2305 , can you please check with the current master? due to the fact you did not merge with the better branch https://github.com/aerospaceresearch/orbitdeterminator/tree/input_format_detection this is now different to what you compared with.
okay?

hornig committed:
Delete orbitjson.txt

a bit big. and also not yet used.
before we add even more files, that at least I don't know yet we will use besides the "test", I would like to use existing ones we have. so this json please do not use yet.

hornig committed:
Delete orbit.json

hornig committed:
Delete orbitiod.txt

hornig committed:
reverted
